### ISSUE TITLE
Fix .circleci/config.yml

### ISSUE THREAD

**Issue by ishikawa999** [ishikawa999/redmine_message_customize#40]
Recent tests have failed with the following error:

```
Error: Database is uninitialized and superuser password is not specified.
       You must specify POSTGRES_PASSWORD to a non-empty value for the
       superuser. For example, "-e POSTGRES_PASSWORD=password" on "docker run".

       You may also use "POSTGRES_HOST_AUTH_METHOD=trust" to allow all
       connections without a password. This is *not* recommended.

       See PostgreSQL documentation about "trust":
       https://www.postgresql.org/docs/current/auth-trust.html
```

**Comment by ishikawa999** [comment]
I will try update orb
https://circleci.com/developer/orbs/orb/agileware-jp/redmine-plugin

**Comment by ishikawa999** [comment]
Merged: #41
